TelepresenceBot/core/src/main/java/com/novoda/tpbot/bot/BotTpService.java

```diff
@@ -0,0 +1,15 @@
+package com.novoda.tpbot.bot;
+
+import com.novoda.tpbot.Direction;
+import com.novoda.tpbot.Result;
+import com.novoda.tpbot.support.Observable;
+
+interface BotTpService {
```

Comment:
It's not very clear how you will use the listen part, but ok for now, let's see in the next PRs